conode/conode.go

```diff
@@ -128,3 +133,29 @@ func checkConfig(c *cli.Context) error {
 	}
 	return check.Config(tomlFileName, c.Bool("detail"))
 }
+
+// Convert toml files from base64-encoded kyes to hex-encoded keys
+func convert64(c *cli.Context) error {
+	scanner := bufio.NewScanner(os.Stdin)
```

Comment:
Please toml-read the file as a `app.GroupToml` and work on that...

And there should be a `private.toml` somewhere - shouldn't that be also converted?

Reply:
Private was always in hex. Thus part of the point of this fix to reduce the confusion.